src/Blocks/BlockFluid.h

```diff
@@ -10,11 +10,11 @@
 class cBlockFluidHandler :
 	public cBlockHandler
 {
-	typedef cBlockHandler super;
+	using Super = cBlockHandler;
```

Comment:
Now we're just deliberately introducing inconsistency :P

Reply:
Already there, and we want classes to start with a capital, so it's actually fixing an inconsistency.